lms/envs/common.py

```diff
@@ -1925,9 +1925,6 @@
     # Credit courses
     'openedx.core.djangoapps.credit',
 
-    # Course teams
```

Comment:
I prefer this comment to the other one ("Teams API"), since this Django app is more than just the API.